docs/nuget-org/package-readme-on-nuget-org.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Package Readme on NuGet.org
+description: Detailed explanation of how Readme files on NuGet.org are rendered and what to do when you run into issues.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 02/23/2021
+ms.topic: conceptual
+ms.reviewer: anangaur
+---
+
+# Package Readme on NuGet.org
+
+[Include a Readme file in your NuGet package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packagereadmefile) to make your package details richer and more informative for your users!
```

Comment:
Which Markdown features are supported? There are a lot Markdown "flavors" each with different features and quirks.